### PR TITLE
BRITECHSRM-223 - Separate assembly for HTTP config

### PR DESCRIPTION
**Pull request by csscottlogic**
This is the first step. The new assembly will be published as a Nuget package.

### CODE REVIEW

**Comment by csscottlogic** [comment]
Sorry about the .sln and .fsproj file diffs. I did this in very granular commits and when I removed Startup.fs from the microservice project, it decided to rewrite the fsproj file. At least that's what I think happened :-(

**Comment by jebbinSCL** [comment]
:shipit:

File: Bristech.Srm.HttpConfig/Serialization.fs [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+module Serialization =
+
+    open System.Web.Http
+    open Newtonsoft.Json
+    open System.Net.Http.Headers
+    open Serilog
+
+    let configure (config: HttpConfiguration) =
+        config.Formatters.XmlFormatter.UseXmlSerializer <- true
```
Comment (dwybourn) [inline Bristech.Srm.HttpConfig/Serialization.fs]: Is this line needed? Are we using xml at all in this project?

**Comment by lclarksl** [comment]
Looks good to me :shipit:

**Comment by pallenSL** [comment]
It does feel wrong to move Startup itself into a library, even if it is currently 100% boilerplate.  I suppose we can suck it and see.

**Comment by csscottlogic** [comment]
Good spot @dwybourn with the XML serialisation. I didn't expend any effort on that because it wasn't a priority, but I suspect we can delete it. I also wondered about whether or not to move Startup, but since it's currently repeated, I thought it'd be OK to do it and then rethink later.

**Pull request merged**